nemo/export/trt_llm/tensorrt_llm_run.py

```diff
@@ -55,7 +56,7 @@ class TensorrtLLMHostContext:
 class TensorrtLLMWorkerContext:
     """The MPI worker side context for TRT LLM inference."""
 
-    decoder: tensorrt_llm.runtime.GenerationSession = None
+    decoder: ModelRunner = None
```

Comment:
Maybe you could also add support for ModelRunnerCpp? It should be a very similar API, and the TRTLLM team tells us that the python backend is really only for testing, and for production the cpp backend should be used